frontend/components/Domain/Recipe/RecipePage/RecipePageParts/RecipePageInstructions.vue

```diff
@@ -339,7 +374,7 @@ export default defineComponent({
     // ===============================================================
     // UI State Helpers
 
-    function validateTitle(title: string | undefined) {
+    function hasSectionTitle(title: string | undefined) {
```

Comment:
This has no direct relevance to the PR, but I renamed this function to be more accurate to what it's doing.